tests/test_kernelization.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import annotations
+import mis.pipeline.kernelization as ker
+import networkx as nx
+
+
+class InitKernelization(ker.Kernelization):
```

Comment:
If I'm kinda nit-picky (euphemism for annoying) you can add this as a fixture and passing the graphs through indirect parametrization.